src/main/java/com/solace/samples/TopicPublisher.java

```diff
@@ -26,57 +30,74 @@
 import javax.jms.Session;
 import javax.jms.TextMessage;
 import javax.jms.Topic;
+
 import com.solacesystems.jms.SolConnectionFactory;
 import com.solacesystems.jms.SolJmsUtility;
 
+/**
+ * Publishes a messages to a topic using Solace JMS 1.1 API implementation.
+ * 
+ * This is the Publisher in the Publish/Subscribe messaging pattern.
+ */
 public class TopicPublisher {
 
-    public void run(String... args) throws Exception {
+    final String SOLACE_VPN = "default";
```

Comment:
This change, moves these two samples away from the rest of the samples in this JMS project (and the style of the rest of the Solace samples) in how the VPN, username and password are handled. At a minimum, I would like to keep the JMS samples internally consistent. So if we change the way arguments are handled in these two samples, I would like to make sure the other JMS samples and associated markdown tutorials are also updated in this pull request.